inference-engine/src/low_precision_transformations/src/network_helper.cpp

```diff
@@ -424,6 +424,32 @@ std::shared_ptr<ngraph::opset1::Multiply> NetworkHelper::optimizeMultipliesAfter
     return nullptr;
 }
 
+bool NetworkHelper::isConvertable(const std::shared_ptr<Node>& node, const element::Type targetType) {
+    const auto constant = as_type_ptr<opset1::Constant>(node);
+    assert(constant);
+
+    const auto values = constant->cast_vector<float>();
+    float min;
+    float max;
+    if (targetType == element::i8) {
+        min = -128.f;
+        max = 127.f;
+    } else if (targetType == element::u8) {
+        min = 0.f;
+        max = 255.f;
+    } else if (targetType == element::i4) {
+        min = -8.f;
+        max = 7.f;
+    } else if (targetType == element::u4) {
+        min = 0.f;
+        max = 15.f;
+    } else {
+        return false;
+    }
```

Comment:
Minor: it is possible to replace this with switch. To do this, you need to use `ngraph::element::Type_t` instead of `ngraph::element`.